src/test/java/org/gridsuite/modification/server/service/BuildTest.java

```diff
@@ -349,6 +350,122 @@ public void runBuildWithEmptyGroupTest() throws Exception {
+                .switchKinds(Arrays.asList(SwitchKind.BREAKER))
+                .couplingDevices(Arrays.asList(CouplingDeviceInfos.builder().busbarSectionId1("vl9_1_1").busbarSectionId2("vl9_2_1").build()))
+                .build().toEntity());
+        // add new Load
+        equipmentsToAdd.add(LoadCreationInfos.builder()
+                .equipmentId("newLoad")
+                .equipmentName("newLoad")
+                .loadType(LoadType.AUXILIARY)
+                .voltageLevelId("vl1")
+                .busOrBusbarSectionId("1.1")
+                .activePower(10.)
+                .reactivePower(20.)
+                .connectionName("vn")
+                .connectionDirection(ConnectablePosition.Direction.TOP)
+                .build().toEntity());
+
+        //add new Line
+        equipmentsToAdd.add(LineCreationInfos.builder()
+                .equipmentId("newLine")
+                .equipmentName("newLine")
+                .seriesResistance(1.0)
+                .seriesReactance(2.0)
+                .shuntConductance1(3.0)
+                .shuntSusceptance1(4.0)
+                .shuntConductance2(5.0)
+                .shuntSusceptance2(6.0)
+                .voltageLevelId1("v1")
+                .busOrBusbarSectionId1("1.1")
+                .voltageLevelId2("v2")
+                .busOrBusbarSectionId2("1B")
+                .connectionName1("cn11")
+                .connectionDirection1(ConnectablePosition.Direction.TOP)
+                .connectionName2("cn22")
+                .connectionDirection2(ConnectablePosition.Direction.TOP)
+                .build().toEntity());
+        // save modifications
+        modificationRepository.saveModifications(TEST_GROUP_ID, equipmentsToAdd);
+
+        // Create build infos
+        BuildInfos buildInfos = new BuildInfos(VariantManagerConstants.INITIAL_VARIANT_ID,
+            NetworkCreation.VARIANT_ID,
+            TEST_REPORT_ID,
+            List.of(TEST_GROUP_ID),
+            List.of(TEST_SUB_REPORTER_ID_1),
+            new HashSet<>());
+
+        // Build variant
+        networkModificationService.buildVariant(TEST_NETWORK_ID, buildInfos);
+
+        // Check if added equipments are indexed in elasticsearch
+        //check voltage level indexation
+        var expectedEquipmentInfos = EquipmentInfos.builder()
+                .networkUuid(TEST_NETWORK_ID)
+                .variantId(NetworkCreation.VARIANT_ID)
+                .id("vl1")
+                .name("vl1")
+                .type(IdentifiableType.VOLTAGE_LEVEL.name())
+                .voltageLevels(Set.of(VoltageLevelInfos.builder().id("vl1").name("vl1").build()))
+                .substations(Set.of(SubstationInfos.builder().id("s1").name("s1").build()))
+                .build();
+        var equipmentInfos = equipmentInfosRepository.findByIdInAndNetworkUuidAndVariantId(List.of("vl1"), TEST_NETWORK_ID, NetworkCreation.VARIANT_ID).get(0);
+        Assertions.assertThat(equipmentInfos)
```

Comment:
Remove `Assertions` and import `assertThat` ?

Reply:
we already have another assertThat from org.hamcrest.MatcherAssert.assertThat